src/javascript/app_2/App/Components/Form/DatePicker/date_picker_wrapper.jsx

```diff
@@ -0,0 +1,102 @@
+import PropTypes                   from 'prop-types';
```

Comment:
I think this component belongs in `Trading/Components/Form/Tradeparams/Duration`?

Reply:
or `Trading/Components/Form/DatePicker` ?